presidio-analyzer/tests/test_recognizer_registry.py

```diff
@@ -49,7 +49,7 @@ def add_custom_pattern_recognizer(self, new_recognizer,
         self.latest_hash = m.digest()
 
     def remove_recognizer(self, name):
-        logging.info("removing recognizer " + name)
+        logging.info("removing recognizer %s", name)
```

Comment:
```suggestion
        logging.info("removing recognizer {}".format(name))
```

Reply:
This logger has it's own internal string formatting, and I'm in favour of using it